README.md

```diff
@@ -6,7 +6,7 @@ Adobe Experience Platform - Messaging is currently in BETA. Use of this code is
 By using the Beta, you hereby acknowledge that the Beta is provided "as is" without warranty of any kind. Adobe shall have no obligation to maintain, correct, update, change, modify or otherwise support the Beta. You are advised to use caution and not to rely in any way on the correct functioning or performance of such Alpha and/or accompanying materials.
 
 ## About this project
-The AEPMessaging extension allows you to send push notification tokens and push notification click-through feedback to the Adobe Experience Platform.
+The AEPMessaging extension enables sending and tracking push notifications in the Adobe Experience Platform.
```

Comment:
`The AEPMessaging extension, along with Adobe Customer Journey Optimizer, enables sending and tracking push notifications in the Adobe Experience Platform iOS SDK. ` ?